app/models/video.rb

```diff
@@ -1,4 +1,4 @@
-class Video < ApplicationRecord
+class Video < ActiveRecord::Base
```

Comment:
Yo could you add this link: https://stackoverflow.com/questions/37359527/why-rails-5-uses-applicationrecord-instead-of-activerecordbase

TLDR ActiveRecord::Base is a Rails 5 thing. They added it in because it's easier to make custom model logic if you're building rails plugins (didn't even know those existed lol).